extensions/guacamole-auth-ldap/src/main/java/org/apache/guacamole/auth/ldap/connection/ConnectionService.java

```diff
@@ -152,11 +191,11 @@
                 }
 
                 // Get associated protocol
-                Attribute protocol = entry.get("guacConfigProtocol");
+                Attribute protocol = entry.get(LDAP_ATTRIBUTE_PROTOCOL);
                 if (protocol == null) {
                     logger.warn("guacConfigGroup \"{}\" is missing the "
-                              + "required \"guacConfigProtocol\" attribute.",
-                            cnName);
```

Comment:
Heh - looks like I'm guilty of brutal over-wrapping myself. ;)

Reply:
Hehe - I'll unwrap things throughout this file.